db/sqlc/academic_years_test.go

```diff
@@ -0,0 +1,71 @@
+package db
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+
+	"github.com/VinCPR/backend/util"
+	"github.com/stretchr/testify/require"
+)
+
+func createRandomAcademicYear(t *testing.T) AcademicYear {
+	RandDate := util.RandomDate()
```

Comment:
golang convention too, since variable starting with uppercase are often exported. So if a function are a variable is not exported, it is preferable to use `randDate`